docs/WindowsBuild.md

```diff
@@ -133,7 +135,10 @@ cmake -B "S:\b\1" ^
   -D LLVM_ENABLE_PDB=YES ^
   -D LLVM_EXTERNAL_CMARK_SOURCE_DIR=S:\cmark ^
   -D LLVM_EXTERNAL_SWIFT_SOURCE_DIR=S:\swift ^
+  -D SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE=S:\swift-syntax ^
   -D SWIFT_PATH_TO_LIBDISPATCH_SOURCE=S:\swift-corelibs-libdispatch ^
+  -D EXPERIMENTAL_STRING_PROCESSING_SOURCE_DIR=S:\swift-experimental-string-processing ^
+  -D SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING=ON ^
```

Comment:
I tend to use `YES`, but that is of little import.

Reply:
It turned out that WinSDK wanted to unconditionally link to `swift_StringProcessing.lib`. So this is the only sane way to provide it. I have not investigated why there was such dependency with `SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING` switched off.

Reply:
Oh, that sounds like we should file an issue for.  I don't think that this is a blocker, but definitely something to investigate (and should be done after we fix things).